lnbits/app.py

```diff
@@ -162,7 +162,7 @@ def signal_handler(signal, frame):
 
     signal.signal(signal.SIGINT, original_sigint_handler)
 
-    logger.info(
+    logger.success(
         f"✔️ Backend {WALLET.__class__.__name__} connected "
```

Comment:
why is there no `+`?

Reply:
you mean for string concatenation? its not needed when inside `()` and if a fn has only one argument

Reply:
TIL
Thanks!